test/test_train_mp_imagenet.py

```diff
@@ -76,12 +91,39 @@
 
 DEFAULT_KWARGS = dict(
     batch_size=128,
-    test_set_batch_size=64,
+    test_set_batch_size=128,
```

Comment:
does this work on v3 too?

Reply:
It works on v3 but I haven't checked on v2. I will revert this.